recipes-soletta/soletta/soletta_git.bb

```diff
@@ -21,11 +22,15 @@ S = "${WORKDIR}/git"
 
 inherit cml1 python3native
 
+PACKAGECONFIG ??= "${@bb.utils.contains('DISTRO_FEATURES','x11', 'x11', '', d)}"
```

Comment:
the condition is not needed. The enumerated features are already meant to be checked for their status. And if enabled (or disabled) then the corresponding elements of `PACKAGECONFIG[feature_name]` apply.